.env

```diff
@@ -5,26 +5,26 @@ SQUARELET_BASE=https://dev.squarelet.com
 
 DC_LOGIN=/accounts/login/squarelet
 DC_LOGOUT=/accounts/logout/
-SQUARELET_SIGNUP=/accounts/signup/?intent=documentcloud&next=
+SQUARELET_SIGNUP="/accounts/signup/?intent=documentcloud&next="
 
 POLL_INTERVAL=5000
-IMAGE_WIDTHS=xlarge:2000,large:1000,normal:700,small:180,thumbnail:60
+IMAGE_WIDTHS="xlarge:2000,large:1000,normal:700,small:180,thumbnail:60"
```

Comment:
Are all of these quote changes necessary?

Reply:
I kept hitting errors running `source .env`. The quotes fixed that.